framework/packages/abstract-ica/src/chain_type.rs

```diff
@@ -0,0 +1,55 @@
+use std::fmt::Display;
+
+use abstract_sdk::std::objects::TruncatedChainId;
+use abstract_sdk::std::registry::*;
+
+#[derive(Debug, Clone, Copy, PartialEq)]
+pub enum ChainType {
+    Evm,
+    Cosmos,
+}
```

Comment:
Shouldn't it be: EVM+CosmwasmVM or Eth+Cosmos?

Reply:
> Not sure what you mean by this

Isn't EVM run SC code on ethereum chains and Cosmwasm VM on cosmos chains?

Reply:
THe `Cosmos` chain type will be for cw-ica-controller (so ICA). We can add CosmWasm in the future as well if we want (and use polytone straight up).